articles/azure/azs-how-move-vm-new-vnet-ps.md

```diff
@@ -76,86 +76,86 @@ $NewNicName = "<output form="newnic" name="result" style="display: inline;">NewN
 $NewPublicIpName = "<output form="newip" name="result" style="display: inline;">NewPublicIp</output>"
 
 # Retrieve VM details
-$VM = Get-AzureRmVM -Name $VMName -ResourceGroupName $RGName
+$VM = Get-AzVM -Name $VMName -ResourceGroupName $RGName
 
 # Check if the VM uses managed or unmanaged disks
 if ($VM.StorageProfile.OsDisk.ManagedDisk) {
     $ManagedDisks = $true
-}
-else {
-    $ManagedDisks = $false
-}
+    }
+    else {
+        $ManagedDisks = $false
+  } 
 
 # Stop the VM
-Stop-AzureRmVM -Name $VMName -ResourceGroupName $RGName -Force
+Stop-AzVM -Name $VMName -ResourceGroupName $RGName -Force
 
 # Get VM current networking details
-$Nic = (Get-AzureRmNetworkInterface | Where-Object -FilterScript { $_.VirtualMachine.Id -like $VM.Id })
-$PublicIp = Get-AzureRmPublicIpAddress | Where-Object -FilterScript { $_.Id -like $Nic.IpConfigurations.PublicIpAddress.Id }
+$Nic = (Get-AzNetworkInterface | Where-Object -FilterScript { $_.VirtualMachine.Id -like $VM.Id })
+$PublicIp = Get-AzPublicIpAddress | Where-Object -FilterScript { $_.Id -like $Nic.IpConfigurations.PublicIpAddress.Id }
 
 # Get new virtual network details
-$NewVNet = Get-AzureRmVirtualNetwork | Where-Object -FilterScript { $_.Name -like $NewNetworkName }
-$NewSubnet = Get-AzureRmVirtualNetworkSubnetConfig -VirtualNetwork $NewVNet -Name $NewSubnetName
+$NewVNet = Get-AzVirtualNetwork | Where-Object -FilterScript { $_.Name -like $NewNetworkName }
+$NewSubnet = Get-AzVirtualNetworkSubnetConfig -VirtualNetwork $NewVNet -Name $NewSubnetName
 
 # Create new VM networking resources
-$NewPublicIp = New-AzureRmPublicIpAddress -Name $NewPublicIpName -Location $NewVNet.Location -ResourceGroupName $NewVNet.ResourceGroupName -Sku $PublicIp.Sku.Name -AllocationMethod $PublicIp.PublicIpAllocationMethod
-$NewIpConfig = New-AzureRmNetworkInterfaceIpConfig -Subnet $NewSubnet -Name "ipconfig2" -Primary -PrivateIpAddress $NewPrivateIp -PublicIpAddress $NewPublicIp
-$NewNic = New-AzureRmNetworkInterface -Name $NewNicName -ResourceGroupName $NewVNet.ResourceGroupName -Location $NewVNet.Location -IpConfiguration $NewIpConfig -NetworkSecurityGroupId $Nic.NetworkSecurityGroup.Id -Force
+$NewPublicIp = New-AzPublicIpAddress -Name $NewPublicIpName -Location $NewVNet.Location -ResourceGroupName $NewVNet.ResourceGroupName -Sku $PublicIp.Sku.Name -AllocationMethod $PublicIp.PublicIpAllocationMethod
+$NewIpConfig = New-AzNetworkInterfaceIpConfig -Subnet $NewSubnet -Name "ipconfig2" -Primary -PublicIpAddress $NewPublicIp
+$NewNic = New-AzNetworkInterface -Name $NewNicName -ResourceGroupName $NewVNet.ResourceGroupName -Location $NewVNet.Location -IpConfiguration $NewIpConfig -NetworkSecurityGroupId $Nic.NetworkSecurityGroup.Id -Force
```

Comment:
What happened with the PrivateIpAddress param?
` -PrivateIpAddress $NewPrivateIp`

Reply:
@ReissJJarvis05 Was there a specific reason why this was removed?